web/pgadmin/static/js/SchemaView/FormView.jsx

```diff
@@ -26,44 +25,53 @@ import { useOnScreen } from '../custom_hooks';
 import { DepListenerContext } from './DepListener';
 import FieldSetView from './FieldSetView';
 
-const useStyles = makeStyles((theme)=>({
-  fullSpace: {
-    padding: 0,
-    height: '100%',
-    overflow: 'hidden',
-  },
-  controlRow: {
-    marginBottom: theme.spacing(1),
-  },
-  nestedTabPanel: {
-    backgroundColor: theme.otherVars.headerBg,
-  },
-  nestedControl: {
-    height: 'unset',
-  },
-  fullControl: {
-    display: 'flex',
-    flexDirection: 'column'
+const StyledBox = styled(Box)((
+  {
+    theme
+  }
+) => ({
+
+  '& .FormView-nestedControl': {
+    height: 'unset !important',
+    '& .FormView-controlRow': {
+      marginBottom: theme.spacing(1),
+    },
+    '& .FormView-nestedTabPanel': {
+      backgroundColor: theme.otherVars.headerBg,
+    }
   },
-  errorMargin: {
+  '& .FormView-errorMargin': {
     /* Error footer space */
     paddingBottom: '36px !important',
   },
-  sqlTabInput: {
-    border: 0,
+  '& .FormView-fullSpace': {
+    padding: '0 !important',
+    height: '100%',
+    overflow: 'hidden',
+    '& .FormView-fullControl': {
+      display: 'flex',
+      flexDirection: 'column',
+      '& .FormView-sqlTabInput': {
+        border: 0,
+      },
+    }
   },
-  nonTabPanel: {
-    padding: 0,
-    background: 'inherit',
+  '&& .FormView-nonTabPanel': {
```

Comment:
&& should be &

Reply:
This is to increase specificity.(To fix background issue for non-tab panel content)